### ISSUE TITLE
Dust emission's soil-moisture gate runs on SPEEDY's root-zone index, not ECHAM's ws/wsmx

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#787]
> **Re-scoped 2026-09-13.** This issue was first written against the CAM/CLM
> dust path in PR #783, which was superseded by the HAMMOZ/Tegen port (#802,
> merged as #804). `wet_threshold_factor` and `soil_water_gwc_scale` no longer
> exist. The underlying gap is unchanged and is now on an *unconditional* code
> path, so the issue stays open; the text below describes the code as merged.

## What is missing

`DustEmissions` uses soil moisture in two places, and both want ECHAM's
`ws/wsmx` — soil water relative to field capacity — which jcm does not carry.

**1. The saturation cut-off, unconditional in every preset**
(`jcm/physics/aerosol/jam/emissions/dust.py`, the `scale` expression):

```python
scale = jnp.where(mask & (wetness <= p.w0), 10.0 * (1.0 - snow) * pot, 0.0)
```

`w0 = 0.99` mirrors `mo_ham_dust.f90`'s `ws/wsmx > 0.99 → no flux`. What is
passed is `forcing.soilw_am`, SPEEDY's **vegetation-weighted root-zone
availability index** (`jcm/data/bc/compile.py`):

```python
rsw   = 1/(swcap + 3*(swcap - swwil))
soilw = min(1, rsw*(swl1 + veg*3*max(0, swl2 - swwil)))
```

**2. The Fécan et al. (1999) threshold correction**, behind
`DustParameters.fecan_moisture` (`k_dust_smst`). Enabled only by `ndust = 2`
(Cheng 2008); the shipped default `ndust = 4` leaves it off, so this path is
dormant today.

## Why it matters

Three mismatches, all pushing the same way — over-suppression of marginal
sources:

1. **It saturates at field capacity, not at porosity.** `soilw_am = 1` already
   means field capacity, so the index compresses the range the cut-off is
   defined against.
2. **It blends the 7-28 cm layer**, which does not dry on the timescale that
   governs saltation; the surface can be emitting while the root zone is not.
3. **It is already vegetation-weighted** (`veg = cvh + 0.8·cvl`), which
   double-counts against the potential-source field, whose `laieff` provenance
   carries vegetation seasonality of its own.

**Current impact is small but unquantified.** Because `soilw_am` is capped at
field capacity by construction it rarely approaches `0.99`, so the cut-off
seldom fires — the science register calls it "close to inert". That is an
argument about the *index*, not evidence that the right field would behave the
same way, and it has not been measured. Deserts sit near zero where none of
this bites; it bites in the marginal sources (Sahel, Australia, central Asia)
where the threshold enters the flux through `(1-r)(1+r)²`.

## Where the hooks are

- The cut-off and the Fécan branch both read `_column_field(forcing,
  "soilw_am", ncols)` in `dust.py`; a real relative-wetness field replaces that
  single read in both.
- ERA5 `swvl1` (0-7 cm volumetric) is already read by the mirror builder
  (`jcm/data/mirror/bundles.py`, `translate_land`) and then *consumed* into the
  index and discarded. Publishing it alongside `soilw_am`, normalised to field
  capacity so it is an ECHAM-like `ws/wsmx` rather than a volumetric content,
  is a bundle field plus a `ForcingData` channel — not a new data source.
- `jcm/data/mirror/SOURCES.md` already lists the ERA5 source.

## What would close it

A relative soil-wetness channel on `ForcingData` fed from ERA5 `swvl1`, both
`dust.py` reads switched to it, and a measurement of how often the cut-off
fires with the correct field versus the index — at minimum on the
release-validation T63 year, since that is the only run long enough to sample
marginal sources across seasons.

Cited from the science register
(`docs/source/science/aerosol.md`, the dust `data` (soil moisture) bullet).
